frontends/PyCDE/test/test_esi.py

```diff
@@ -156,13 +156,13 @@ def unwrap_and_pad(ports, input_channel: ChannelSignal):
 # CHECK:         %c0_i224 = hw.constant 0 : i224
 # CHECK:         [[r0:%.+]] = comb.concat %c0_i224, %Consumer.loopback_out : i224, i32
 # CHECK:         [[r1:%.+]] = comb.extract %trunk_in from 0 {sv.namehint = "trunk_in_0upto32"} : (i256) -> i32
-# CHECK:         %Producer.int_out, %Producer.int_out_valid, %Producer.loopback_in_ready = hw.instance "Producer" sym @Producer @Producer<__INST_HIER: none = #hw.param.expr.str.concat<#hw.param.decl.ref<"__INST_HIER">, ".Producer">>(clk: %clk: i1, loopback_in: [[r1]]: i32, loopback_in_valid: %trunk_in_valid: i1, int_out_ready: %Consumer.int_in_ready: i1) -> (int_out: i32, int_out_valid: i1, loopback_in_ready: i1)
-# CHECK:         %Consumer.loopback_out, %Consumer.loopback_out_valid, %Consumer.int_in_ready = hw.instance "Consumer" sym @Consumer @Consumer<__INST_HIER: none = #hw.param.expr.str.concat<#hw.param.decl.ref<"__INST_HIER">, ".Consumer">>(clk: %clk: i1, int_in: %Producer.int_out: i32, int_in_valid: %Producer.int_out_valid: i1, loopback_out_ready: %trunk_out_ready: i1) -> (loopback_out: i32, loopback_out_valid: i1, int_in_ready: i1)
+# CHECK:         %Producer.loopback_in_ready, %Producer.int_out, %Producer.int_out_valid = hw.instance "Producer" sym @Producer @Producer<__INST_HIER: none = #hw.param.expr.str.concat<#hw.param.decl.ref<"__INST_HIER">, ".Producer">>(clk: %clk: i1, loopback_in: [[r1]]: i32, loopback_in_valid: %trunk_in_valid: i1, int_out_ready: %Consumer.int_in_ready: i1) -> (loopback_in_ready: i1, int_out: i32, int_out_valid: i1)
```

Comment:
There's a lot of copy pasted output in the ESI tests. I don't think there's any way around changing these specific tests with this specific change, but in general, it would be good to use more regexes and avoid putting unnecessary IR into the tests. For example, the whole `<__INST_HIER: none = ...>` parameter lists probably don't need to be checked on every test here.

Reply:
yep. I agree.

Reply:
This one is particularly egregious since I didn't even remove the output file attribute, which isn't necessary for this test!